R/derive_vars_query.R

```diff
@@ -1,5 +1,8 @@
 #' Derive Query Variables
 #'
+#' **Note:** This function exists to work with derive_vars_query, to execute
+#' the first step of derive_vars_query separately for metadata tracking
+#'
```

Comment:
The `@return` section needs to be updated because not all variables of the input dataset are included in the output dataset.

`@family` and `@keywords` should be changed to `utils_help` because the function is not a derivation.